pastel/act_ticket.go

```diff
@@ -0,0 +1,24 @@
+package pastel
+
+// ActivationTicketType represents a type of the activation ticket.
+type ActivationTicketType string
+
+// List of types of activation ticket.
+const (
+	ActTicketAll       ActivationTicketType = "all"
+	ActTicketAvailable ActivationTicketType = "available"
+	ActTicketSold      ActivationTicketType = "sold"
+)
```

Comment:
be consistent, use `Act` or  `Activation` everywhere.
I personally prefer `Act` to reflect API method